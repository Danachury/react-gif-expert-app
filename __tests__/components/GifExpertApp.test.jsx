import { render, screen } from '@testing-library/react'
import GifExpertApp from '../../src/GifExpertApp.jsx'

describe('Tests in <GifExpertApp />', () => {

    test('should match with snapshot', () => {
        const { container } = render(<GifExpertApp/>)
        expect(screen.getByRole('heading', { level: 1 }).innerHTML).toBe('GifExpertApp')
        expect(container).toMatchSnapshot()
    })
})
