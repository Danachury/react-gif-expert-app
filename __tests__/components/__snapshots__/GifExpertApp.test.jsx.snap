// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <GifExpertApp /> should match with snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Search gifs"
      type="text"
      value=""
    />
  </form>
</div>
`;
